cypress/integration/tests/developer/applicationinventory/analysis/upload_bynary_analysis_migrator.test.ts

```diff
@@ -0,0 +1,119 @@
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteAllBusinessServices,
+    deleteApplicationTableRows,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    hasToBeSkipped,
+    login,
+    preservecookies,
+    resetURL,
+    writeGpgKey,
+} from "../../../../../utils/utils";
+import { Proxy } from "../../../../models/administrator/proxy/proxy";
+import { Analysis } from "../../../../models/developer/applicationinventory/analysis";
+import {SEC} from "../../../../types/constants";
+
+describe("Upload Binary Analysis", { tags: "@tier1" }, () => {
+    before("Login", function () {
+        // Prevent hook from running, if the tag is excluded from run
+        if (hasToBeSkipped("@tier1")) return;
+
+        // Perform login
+        login();
+        deleteApplicationTableRows();
+
+        //Disable all proxy settings
+        Proxy.disableAllProxies();
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        preservecookies();
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+
+        // Interceptors
+        cy.intercept("POST", "/hub/application*").as("postApplication");
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+    });
+
+    afterEach("Persist session", function () {
+        // Reset URL from report page to web UI
+        resetURL();
+    });
+
+    after("Perform test data clean up", function () {
+        // Prevent hook from running, if the tag is excluded from run
+        deleteApplicationTableRows();
+        deleteAllBusinessServices();
+        writeGpgKey("abcde");
+    });
+
+    it("Upload Binary Analysis", function () {
+        const application = new Analysis(
+            getRandomApplicationData("uploadBinary"),
+            getRandomAnalysisData(this.analysisData[4])
+        );
+        application.create();
+        cy.wait("@getApplication");
+        cy.wait(2 * SEC);
+        // No credentials required for uploaded binary.
+        application.analyze();
+        application.verifyAnalysisStatus("Completed");
+        application.openreport();
+        application.validateStoryPoints();
+    });
+
+    it("Custom rules with custom targets", function () {
+        // Automated https://issues.redhat.com/browse/TACKLE-561
+        const application = new Analysis(
+            getRandomApplicationData("customRule_customTarget"),
+            getRandomAnalysisData(this.analysisData[5])
+        );
+        application.create();
+        cy.wait("@getApplication");
+        cy.wait(2 * SEC);
+        // No credentials required for uploaded binary.
+        application.analyze();
+        application.verifyAnalysisStatus("Completed");
```

Comment:
Why is this file changed ?

Reply:
Removed from commit, I don't know how this file got there